### DISCUSSION TITLE
View partial record

### DISCUSSION THREAD

**Discussion by joranmulderij** [pocketbase/pocketbase#1761] [Q&A] [answered]
Is it possible to read only a single field from a record?

**Comment by ganigeorgiev** [comment, marked as answer]
No, this currently is not supported.
Option to filter the response fields using `?fields=field1,fiedl2` query parameter is in my todo and eventually will be added in the future but for now is a very low priority.
A new "view" collection type that will allow creating a readonly collection with a custom query or a subset of existing collection fields will be available in the next v0.13/v0.14 release.
